programmers2/모의고사.js

```diff
@@ -0,0 +1,19 @@
+function solution(answers) {
+    const a = [1,2,3,4,5];
+    const b = [2,1,2,3,2,4,2,5];
+    const c = [3,3,1,1,2,2,4,4,5,5];
```

Comment:
코딩테스트에서는 a, b, c 처럼 선언해주는 것이 빠르게 문제를 푸는데 도움이 될 순 있습니다.
다만, 개발은 협업입니다.
a, b, c처럼 변수를 선언했다가는 사수한테 뺨다구 5만대 기대하시면 됩니다ㅋㅋㅋㅋㅋㅋ
실제 접근할 때는 ```patterns[0]``` 으로 접근해서 값을 비교해주시면 됩니다.

추가로, 최근에는 코딩테스트에서도 변수를 어떻게 선언하며 개발하는지 보는 회사들이 늘고 있습니다.
알아두시고, a, b, c와 같은 무자비한 변수명은 피하는 것이 좋아요 ^^
```suggestion
    const patterns = [
        [1,2,3,4,5],
        [2,1,2,3,2,4,2,5],
        [3,3,1,1,2,2,4,4,5,5]
     ];
```